k8s/build_image.sh

```diff
@@ -27,7 +27,7 @@ fi
 
 echo "Building in $PWD..."
 
-VERSION=$(./SCYLLA-VERSION-GEN)
-PACKAGE_NAME="scylladb/scylla-machine-image-k8s-$CLOUD_PROVIDER:$VERSION"
+VERSION="k8s-${CLOUD_PROVIDER}-node-setup-0.0.2"
+IMAGE_REF="scylladb/scylla-machine-image:${VERSION}"
 
-docker build . -f k8s/Dockerfile --build-arg cloud_provider=$CLOUD_PROVIDER -t $PACKAGE_NAME
\ No newline at end of file
+docker build . -f k8s/Dockerfile --build-arg cloud_provider=${CLOUD_PROVIDER} -t ${IMAGE_REF}
```

Comment:
nit
```suggestion
docker build -f k8s/Dockerfile --build-arg "cloud_provider=${CLOUD_PROVIDER}" -t "${IMAGE_REF}" .
```

Reply:
done

Reply:
missing the `IMAGE_REF` quotes still but I don't want to hold this fix

Reply:
done